### ISSUE TITLE
Empty doc pages with missing links to subtopics

### ISSUE THREAD

**Issue by Wryhder** [asterisk/documentation#87]
**Expected behaviour:** Clicking on a high-level topic or a label used to organize related subtopics should open an overview page with some content (e.g, _Fundamentals > Asterisk Architecture_) or load the first sub-item (e.g, _Getting Started > Beginning Asterisk_).

**Actual behaviour:** There are several empty overview pages.

I'd like to suggest including links to the sub-topics or sub-items on the overview page pending when the overview content (if planned) is created. 

![empty page missing links to subtopics-fundamentals](https://github.com/asterisk/documentation/assets/21000445/8d8dc577-9168-4a9d-a482-840378f2d827)

**TODO:** Update issue with links to all empty overview pages.

**Comment by gtjoseph** [comment]
The page for [Asterisk Architecture](https://docs.asterisk.org/Fundamentals/Asterisk-Architecture/) does have content...
![image](https://github.com/asterisk/documentation/assets/1205666/c4773cb6-9914-4bd6-ae64-95a31d21dd69)

Adding links to sub-topics in the overview content pages is tricky because the navigation is generated dynamically by mkdocs.  I'll see if there's some way we can handle it.

**Comment by Wryhder** [comment]
> The page for [Asterisk Architecture](https://docs.asterisk.org/Fundamentals/Asterisk-Architecture/) does have content... ![image](https://private-user-images.githubusercontent.com/1205666/330077384-c4773cb6-9914-4bd6-ae64-95a31d21dd69.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU2MTQ2MDgsIm5iZiI6MTcxNTYxNDMwOCwicGF0aCI6Ii8xMjA1NjY2LzMzMDA3NzM4NC1jNDc3M2NiNi05OTE0LTRiZDYtYWU2NC05NWEzMWQyMWRkNjkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTNUMTUzMTQ4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZGUyMTZlZmZkYjMzYTJlOTQyNjMyYTBiMDAxYWNlMDJhNDQwNmI5NzhlNDVhZTU0MDFmN2E0ZGMwMzNlOTkwMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.-BtKOd4YkEioa7HbYUoX8eUg-Cx8DGjXlrTZJfFXUQY)

Yes, I included this as an example of an overview page with some content. I suppose I didn't express that clearly.
> 
> Adding links to sub-topics in the overview content pages is tricky because the navigation is generated dynamically by mkdocs. I'll see if there's some way we can handle it.

What should go into the overview docs? Will a summary of the ideas in the subsections suffice (with relevant links to each subsection within the overview, along with footer links to the previous and next page of the current page)?

**Comment by gtjoseph** [comment]
@Wryhder Sorry for the delay.  If you still want to work on it, why don't you create one or two as an example so we can see what you have in mind and submit a PR.  We can go from there.

**Comment by ipaqmaster** [comment]
Trying to use this documentation has been a pretty frustrating experience to say the least. Most of the pages just say under construction and have blank content. Where's the actual documentation?

**Comment by Wryhder** [comment]
@gtjoseph Apologies. I'd missed your previous response.  I'll look into creating an example and get back to you within the next few weeks.

**Comment by Wryhder** [comment]
@ipaqmaster I can't take a look right now, but wasn't it mostly the overview pages that have missing content?

If you use the left sidebar navigation and expand the link categories with the arrows, there's a lot of existing documentation. Unless there's something specific not covered?

(Not sure where you can get a faster response from the team.)
